zipfile2/tests/test__zipfile.py

```diff
@@ -195,6 +195,7 @@ def test_multiple_archives_write(self):
     def test_multiple_archives_writestr(self):
         # Given
         zipfile = os.path.join(self.tempdir, "foo.zip")
+        target = os.path.join(self.tempdir, "file.py")
```

Comment:
This looks like a candidate for a separate PR. It's not related to the `LeanZipFile` change.